helics/cli.py

```diff
@@ -283,6 +283,14 @@ def run(path, silent, connect_server, no_log_files, no_kill_on_error):
     if helics_server_available:
         fetch("/runner/file/name", {"name": os.path.basename(path_to_config)})
         fetch("/runner/file/folder", {"folder": os.path.dirname(path_to_config)})
+    
+    # Default to logging in the same location as the config file; this is the 
+    # historical behavior of the cli. If there's a "logging_path" in the runner
+    # JSON, use that path    
+    if "logging_path" in config.keys():
+        logging_path = config["logging_path"]
+    else:
+        logging_path = path
```

Comment:
I think this got added in two different commits? This change was already in main, so it's duplicated right below this.

Reply:
Ok, removed the duplicate lines.